react-native-libraries.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "githubUrl": "https://github.com/pachun/simple-expo-update",
+    "npmPkg": "@pachun/simple-expo-update",
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true
+  },
```

Comment:
Let's move the new entry at the end of file:
* https://github.com/react-native-community/directory?tab=readme-ov-file#how-do-i-add-a-library

Otherwise looks good! 👍

Reply:
@Simek Oops! I did it right once before 🫠 Got distracted tryna figure out how to sync my account's fork of this repo since the last PR I made and followed the steps less closely this time. PR should be fixed now. Thanks for the help!